lib/merge.js

```diff
@@ -135,9 +135,6 @@ function doMerge(tree, path, dontExpand) {
           }
           continue;
         }
-        if (!item.ids) {
-          continue;
-        }
```

Comment:
Heh all this code is fairly hairy but its well tested (https://github.com/pouchdb/pouchdb/blob/master/tests/unit/test.merge.js), if we arent hitting this then I think we are safe.